src/renderer/api-decorator.js

```diff
@@ -426,11 +426,14 @@ limitations under the License.
         let webContentsId = getWebContentsId();
         // Reset state machine values that are set through synchronous handshake between native WebContent lifecycle observers and JS
         options.openfin = true;
-        options.preload = 'preload' in options ? options.preload : winOpts.preload;
+
+        // Force window to be a child of its parent application.
         options.uuid = winOpts.uuid;
+
         // Apply parent window background color to child window when child
         // window background color is unspecified.
         options.backgroundColor = options.backgroundColor || winOpts.backgroundColor;
```

Comment:
Question about this existing code:
Why is this needed here? Is `Window.create` being called? (How, btw?) If it is, note that it already inherits from main window options (see [above](https://github.com/HadoukenIO/core/pull/177/files#diff-d5752a5b2c3b7ce27c77d78f1b9278a6L398)).